libs/sdk-ui-tests-e2e/README.md

```diff
@@ -146,3 +146,18 @@ and run with (by default runs in visual mode)
 ```
 CYPRESS_HOST=http://localhost:9500 yarn run-integrated
 ```
+
+### Parallelization
+
+By default, cypress is running in serial mode. To enable parallelization, [Currents](https://currents.dev/readme/getting-started/you-first-cypress-run) is integrated into cypress.
+To use Currents, `RUN_PARALLEL` must be set to `true` and following environemnts must be set
+
+```
+export CURRENTS_PROJECT_ID
+export CYPRESS_RECORD_KEY
+export CURRENTS_CI_BUILD_ID
+```
+
+-   `CURRENTS_PROJECT_ID` is unique for each UI-app respository, for development purpose is `4wrwOa`
```

Comment:
This is not correct dev PROJECT_ID, please check https://docs.google.com/document/d/1tRJhhwiuFX5PvASGRSgqp1HJfMSK2aCS7CQwGi_aYXM/edit#heading=h.rhybfzm6mz3x

Reply:
updated